mlflow/pyfunc/__init__.py

```diff
@@ -372,6 +372,19 @@ def predict(self, data: PyFuncInput) -> PyFuncOutput:
             data = _enforce_schema(data, input_schema)
         return self._predict_fn(data)
 
+    def unwrap_python_model(self, unwrap_metadata: bool = False):
+        """
+        Unwrap the underlying Python model object, and it's metadata (optionally).
+
+        :param unwrap_metadata: (optional) If True, return a tuple of the
+        underlying Python model object and metadata loaded from the MLmodel file.
```

Comment:
mlflow/pyfunc/__init__.py:docstring of mlflow.pyfunc.PyFuncModel.unwrap_python_model:4: WARNING: Field list ends without a blank line; unexpected unindent.
(start of the new line of a comment needs to align with the preceding line's text following the `:` of an arg definition)

Reply:
Removed `unwrap_metadata` param